bookwyrm/views/books/series.py

```diff
@@ -0,0 +1,60 @@
+""" books belonging to the same series """
+from django.views import View
+from django.shortcuts import get_object_or_404
+from django.template.response import TemplateResponse
+
+from bookwyrm.views.helpers import is_api_request
+from bookwyrm import models
+
+
+# pylint: disable=no-self-use
+class BookSeriesBy(View):
+    """book series by author"""
+
+    def get(self, request, author_id):
+        """lists all books in a series"""
+        series_name = request.GET.get("series_name")
+
+        if is_api_request(request):
+            pass
+
+        author = get_object_or_404(models.Author, id=author_id)
+
+        results = models.Edition.objects.filter(authors=author, series=series_name)
+
+        # when there are multiple editions of the same work, pick the closest
+        editions_of_work = results.values_list("parent_work__id", flat=True).distinct()
```

Comment:
If there is an edition with a series number, but a "closer" one without a series number, we should ideally prefer to use the one with the number.

Reply:
The latest change should sort the editions with a series number first, then the top ranking edition